net-libs/enet/enet-1.3.0.recipe

```diff
@@ -8,14 +8,15 @@ lobbying, server discovery, encryption, or other similar tasks that are \
 particularly application specific so that the library remains flexible, \
 portable, and easily embeddable."
 HOMEPAGE="http://enet.bespin.org"
```

Comment:
Should be:
```
HOMEPAGE="http://enet.bespin.org/"
```
(trailing slash added)
BTW, even if most browsers hide the trailing slash when it is after the domain name, a correct URI with no path has to have a trailing slash.
On Firefox, for example, if you set "browser.urlbar.trimURLs" to false you will see the trailing slash.